vips/transform_test.go

```diff
@@ -151,13 +151,19 @@ func TestTransform_MaxScale(t *testing.T) {
 
 func TestTransform_Scale_Alpha_1(t *testing.T) {
 	goldenTest(t, resources+"logo_alpha.png", func(tx *Transform) {
-		tx.Scale(0.4)
+		tx.Scale(0.2)
 	})
 }
 
 func TestTransform_Scale_Alpha_2(t *testing.T) {
 	goldenTest(t, resources+"logo_alpha_2.png", func(tx *Transform) {
-		tx.Scale(0.4)
+		tx.Scale(0.2)
```

Comment:
Why did we change the scale in the test?

Reply:
simulating real life